dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task210816DeInodeRelationship.java

```diff
@@ -0,0 +1,88 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import com.dotmarketing.util.Logger;
+import io.vavr.control.Try;
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * 1. Adds `mod_date` column to `relationship` table
+ * 2. Copies `inode.idate` to new `relationship.mod_date`
+ * 3. Drops FK from `relationship` to `inode`
+ */
+public class Task210816DeInodeRelationship extends AbstractJDBCStartupTask {
```

Comment:
Shouldn't the UT also delete all the inodes entries that are now "orphanage"?

Reply:
Done